applications_tests/gls_sharp_navier_stokes_3d/pp_lubrication_test.output

```diff
@@ -0,0 +1,82 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       1600
+   Number of degrees of freedom: 8228
+   Volume of triangulation:      1600
+Initial refinement around IB particles - Step : 1 of 3
+   Number of active cells:       2160
+   Number of degrees of freedom: 11244
+   Volume of triangulation:      1600
+Initial refinement around IB particles - Step : 2 of 3
+   Number of active cells:       4876
+   Number of degrees of freedom: 24588
+   Volume of triangulation:      1600
+Initial refinement around IB particles - Step : 3 of 3
+   Number of active cells:       4876
+   Number of degrees of freedom: 24588
+   Volume of triangulation:      1600
+
+*****************************************************************
+Transient iteration : 1        Time : 0.025    Time step : 0.025    CFL : 0       
+*****************************************************************
+particule 0 position 5.0000005753691994 12.72541802259715 4.9998822284914377
+particule 0 velocity 2.3014767991157072e-05 -0.98327909611401321 -0.0047108603424777691
+particule 1 position 5.0000049478375965 11.030914655716716 4.9998942235652279
+particule 1 velocity 0.00019791350384434301 1.2365862286686038 -0.0042310573908838174
```

Comment:
This number of significant digits thing is going out of hand. You'll need to fix this in a subsequent PR.